src/Security/Http/Logout/ContextInvalidationLogoutHandler.php

```diff
@@ -24,7 +24,7 @@
  * This handler is deprecated because it never did what it was supposed to do. The session is already invalidated by the SessionLogoutHandler
  * which is always the first logout handler executed
  */
-final class ContextInvalidationLogoutHandler implements LogoutHandlerInterface
+class ContextInvalidationLogoutHandler implements LogoutHandlerInterface
```

Comment:
why do we need to un-final this one?

Reply:
Oh, this was definitely not intended. Thanks for noticing. Just reverted with commit https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/pull/621/commits/2a8f76d0e5bfabd878f3b102d2e1ac440761df52

This class will then be removed with PR https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/pull/618